nflow-engine/src/main/java/com/nitorcreations/nflow/engine/workflow/instance/WorkflowInstance.java

```diff
@@ -43,6 +43,12 @@
   public final Integer executorId;
 
   /**
+   * The id of the workflow that created the hierarchy of workflow where this sub workflow belongs to.
+   * Null for workflows without children and workflows that are root of hierarchy.
```

Comment:
Shouldn't it be "Null for workflows that are the root of the hierarchy"? Child workflows may not have their own children but they may belong to a hierarchy so they should have root id set.